Lib/fontbakery/reporters/terminal.py

```diff
@@ -369,6 +369,9 @@ def __init__(self, collect_results_by=None
         # status after ENDCHECK.
         self._render_async = self.is_async or check_threshold is not None
 
+    def write(self):
```

Comment:
Maybe move this into `FontbakeryReporter`, or, there, do something like this:

```py
def write(self):
    if self.output_file is not None:
       raise NotImplementedError(f'{type(self)} does not implement the "write" method, '
                                  'but it has an "output_file".')
    # do nothing
```



FontbakeryReporter